app/src/main/java/com/kabouzeid/gramophone/ui/activities/AlbumDetailActivity.java

```diff
@@ -327,6 +388,7 @@ public void setStatusbarColor(int color) {
     private void setAlbum(Album album) {
         this.album = album;
         loadAlbumCover();
+        loadWiki();
```

Comment:
@Materight is right, should behave the same as the artist bio. However in this case we should rename the preference to auto download artist info to something like "Auto download meta data" or so.